.github/workflows/pull_requests.yaml

```diff
@@ -125,3 +125,36 @@ jobs:
         env:
           PREVIEW_TAG: "${{ steps.push-container.outputs.PREVIEW_TAG }}"
         run: ./ci/generate_docker_report.sh
+
+  Push-Registry:
+    needs: [Acceptance-Tests, Modules-Acceptance-Tests, Unit-Tests, Integration-Tests, Vulnerability-Scanning]
+    runs-on: ubuntu-latest-8-cores
+    if: ${{ !github.event.pull_request.head.repo.fork }}  # no PRs from fork
+    permissions:
+      packages: write
+      contents: read
+    env:
+      IMAGE_NAME: weaviate
+    steps:
+      - uses: actions/checkout@v3
+      - name: Build image
+        run: docker build . --file Dockerfile --tag $IMAGE_NAME
```

Comment:
I assume we'd be limited to `amd64` without the multi-env build. I also assume this doesn't matter because we would only ever use those images on CI which is in turn `amd64` based. Correct?

Reply:
exactly! We can maybe add arm64 later, but at first I want to migrate other repos to use the github registry to remove uasge of the DOCKERHUB secrets from the client repos. This makes it simpler to let externals contribute